pkgs/tools/admin/fioctl/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "fioctl";
+  version = "0.14.1";
+
+  src = fetchFromGitHub {
+    owner = "foundriesio";
+    repo = "fioctl";
+    rev = "v${version}";
+    sha256 = "1jbj2w2s78wcnrwyr80jyc11ipjysv5aab3486kphx8ysvvgcwfs";
+  };
+
+  vendorSha256 = "1a3x6cv18f0n01f4ac1kprzmby8dphygnwsdl98pmzs3gqqnh284";
+
+  runVend = true;
+
+  buildFlagsArray = ''
+    -ldflags=
+      -X github.com/foundriesio/fioctl/subcommands/version.Commit=${src.rev}
+  '';
+
+  meta = with lib; {
+    description = "A simple CLI to manage your Foundries Factory ";
+    homepage = https://github.com/foundriesio/fioctl;
+    license = licenses.asl20;
+    maintainers = with maintainers; [ nixinator matthewcroughan ];
+    platforms = platforms.linux ++ platforms.darwin;
```

Comment:
```suggestion
```

set by buildGoModule

Reply:
https://nixos.org/manual/nixpkgs/stable/#ssec-go-modules

This documentation suggests setting of `platforms =` with buildGoModule. Should we update the manual to reflect this?

Reply:
Is this because URI was formerly its own type? https://nixos.wiki/wiki/Nix_Expression_Language#Types